src/sbvr-api/sbvr-utils.ts

```diff
@@ -1158,6 +1162,84 @@ const runODataRequest = (req: Express.Request, vocabulary: string) => {
 		api[vocabulary].logger.log('Parsing', req.method, req.url);
 	}
 
+	if (req.url === `/${vocabulary}/$batch`) {
+		const { requests } = req.body as { requests: uriParser.UnparsedRequest[] };
```

Comment:
Can you separate the contents of this `if` into a separate function, eg `parseBatch`